source/usr/lib/mcvirt/virtual_machine/virtual_machine.py

```diff
@@ -850,17 +908,22 @@ def getAvailableNodes(self):
     def ensureRegisteredLocally(self):
         """Ensures that the VM is registered locally, otherwise an exception is thrown"""
         if (not self.isRegisteredLocally()):
-            raise VmRegisteredElsewhereException('The VM \'%s\' is registered on the remote node: %s' %
-                                                 (self.getName(), self.getNode()))
+            raise VmRegisteredElsewhereException(
+                'The VM \'%s\' is registered on the remote node: %s' %
+                (self.getName(), self.getNode()))
 
     def getVncPort(self):
         """Returns the port used by the VNC display for the VM"""
         # Check the user has permission to view the VM console
-        self.mcvirt_object.getAuthObject().assertPermission(Auth.PERMISSIONS.VIEW_VNC_CONSOLE, self)
+        self.mcvirt_object.getAuthObject().assertPermission(
+            Auth.PERMISSIONS.VIEW_VNC_CONSOLE,
+            self)
 
         if (not self.getState()):
             raise MCVirtException('The VM is not running')
-        domain_xml = ET.fromstring(self._getLibvirtDomainObject().XMLDesc(libvirt.VIR_DOMAIN_XML_SECURE))
+        domain_xml = ET.fromstring(
+            self._getLibvirtDomainObject().XMLDesc(
+                libvirt.VIR_DOMAIN_XML_SECURE))
 
         if (domain_xml.find('./devices/graphics[@type="vnc"]') == None):
```

Comment:
I am seeing the following pep8 error here:

    ./source/usr/lib/mcvirt/virtual_machine/virtual_machine.py:928:64: E711 comparison to None should be 'if cond is None:'
            if (domain_xml.find('./devices/graphics[@type="vnc"]') == None):
                                                                   ^